client/src/pages/footer/styles.ts

```diff
@@ -0,0 +1,129 @@
+import styled from 'styled-components';
+
+export const Rodape = styled.footer`
+    position: absolute;
+    width: 100%;
+    height: 360px;
+    left: 0px;
+    right: 0px;
+    top: 5224px;
+    background: linear-gradient(180deg, #303030 0%, #2D2D2D 28.65%, #242424 100%);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
```

Comment:
não é interessante que usemos o position absolute, nesse caso o melhor é retira-lo, quando colocamos um componente abaixo do outro no app eles serão renderizados exatamente nessa ordem, então se o footer for o último ele ja estará embaixo.